lib/php-google-oauth/auth/Google_OAuth2.php

```diff
@@ -110,9 +110,8 @@ public function authenticate($service, $code = null) {
         if ($decodedResponse != null && $decodedResponse['error']) {
           $response = $decodedResponse['error'];
         }
-		ns_google_sheets_connector::ns_debug_log("Error fetching OAuth2 access token, message: '$response' :: {$request->getResponseHttpCode()}");
-		
-        throw new Google_AuthException("Error fetching OAuth2 access token, message: '$response'", $request->getResponseHttpCode());
+		  echo "<span style='color:red'>".("You have entered an incorrect access code")."</span>";
+      die();
```

Comment:
@Abhishilawat This change needs to be in our code, don't change the code from Google's library.